packages/thumbprint-font-face/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Changed
+
+-   [Minor] Change font path to point to subsetted version of font.
```

Comment:
```suggestion
-   [Major] Change font path to point to subsetted version of font.
```

Would technically be a breaking change since it requires the font to exist on `$thumbprint-font-url`. Won't actually require an upgrade for internal Thumbtack users if we put the font on our server.